OpenEdXMobile/src/main/java/org/edx/mobile/util/CalendarUtils.kt

```diff
@@ -418,4 +420,19 @@ object CalendarUtils {
     ): String {
         return "${environment.config.platformName} - $courseName"
     }
+
+    /**
+     * Custom Loading Dialog to show while creating/updating Calendar Events
+     * @param context - current [Context] of the application
+     * @param layoutInflater
+     */
+    @JvmStatic
+    fun getLoadingDialog(context: Context, layoutInflater: LayoutInflater): AlertDialog {
```

Comment:
IMO, Instead of this, we can extent the `AlertDialogFragment` by :
```   
    public static AlertDialogFragment newInstance(@StringRes int titleResId, final @LayoutRes int viewId) {
        final AlertDialogFragment fragment = new AlertDialogFragment();
        final Bundle args = new Bundle();
        args.putInt(ARG_TITLE_RES, titleResId);
        args.putInt(ARG_DIALOG_VIEW, viewId);
        fragment.setArguments(args);
        return fragment;
    }
```    

And this in `onCreateDialog`:
```
    final int layoutResId = args.getInt(ARG_DIALOG_VIEW, 0);
        if (layoutResId != 0) {
            View customView = getActivity().getLayoutInflater().inflate(R.layout.alert_dialog_progress, null);
            alertDialog.setView(customView);
        }
```